skim_2links.clj

```diff
@@ -0,0 +1,70 @@
+(config
+    (text-field
+        :name "clientId" 
+        :label "Enter clientId"
+        :placeholder "Enter Client Id"
+        :required true
+        (api-config-field)
+    )
+    (password-field
+        :name "clientSecret" 
+        :label "Enter clientSecret"
+        :placeholder "Enter client secret"
+        :required true
+        (api-config-field :masked true)
+    )
+    (text-field
+        :name "PublisherId" 
+        :label "Publisher Id"
+        :placeholder "Enter Publisher Id"
+        :required true
+    )
+    (oauth2/authorization-code-flow-client-credentials
```

Comment:
it should be refresh-token-wit- client-credentials, we should get it fixed in provided docs